src/lib/y2users/linux/users_writer.rb

```diff
@@ -328,7 +328,10 @@ def commit_config(user)
       def exist_user_home?(user)
         return false unless user.home&.path
 
-        File.exist?(user.home.path)
+        Yast::Execute.on_target!("/usr/bin/stat", user.home.path)
+        true
```

Comment:
Another option would be 

```suggestion
        Yast::Execute.on_target!("/usr/bin/stat", "--printf", "%F", user.home.path, stdout: :capture) == "directory"
```

but I have doubts if it is a good idea. What do you think?

Reply:
BTW, the use of `Yast::Execute` instead of `Yast::FileUtils.Exists?` is intentional, after discussed it with @joseivanlopez

Reply:
I prefer the suggested version with "directory".

And please add a comment to the code explaining why you decided to not use `Yast::FileUtils.Exists?` "After discussing it with Iván" is a bit vague. ;-)

Reply:
That being said, I think the most YaSTie way for checking for a file/directory in the target system is using the target agent: https://github.com/yast/yast-core/blob/master/doc/systemagent.md

Reply:
Thanks for the comment @ancorgs. Yes, it was a bit vague from my side to not give a more elaborated explanation :wink: Summarizing we (specially me) thought that `Yast::Execute` somehow obsoletes SCR target agents. But as you explained in the YaST  IRC channel:

> **\<ancorgs\>** to a certain extend, Yast::Execute substitutes/obsoletes target.bash and target.bash_output... but not the rest of the utilities in the target agent

So, let's change it and use `Yast::FileUtils.Exists?`.

Reply:
> let's change it and use `Yast::FileUtils.Exists`

Or even [`Yast::FileUtils.IsDirectory`](https://github.com/yast/yast-yast2/blob/25a8bb990c4b9e81d2af2c4a699288dd95f90900/library/general/src/modules/FileUtils.rb#L61-L76)